documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,145 @@
+      2. Twin competitions are obliged to clearly and publicly state their condition of twin competitions and their special participation terms.
+         1. A competitor is only permitted to participate in one of the twin competitions.
+            1. If a competitor competes at more than one of the involved twin competitions, only the results from the competition that they compete in first will be kept.
+3. WCA Delegate(s)
+4. Organizer(s)
+   1. Organizers can be specific persons and/or organizations.
+5. Competitor limit
+   1. The competitor limit can be on either a "first come first serve" basis or based upon qualification results or rankings in the WCA world rankings of a previously announced calendar date.
+      1. Qualification for an event must be based only on results from the same event.
+   2. If the competitor limit for a competition or an event is reached before the competition, the organization team must not approve any additional registrations for the competition or the event.
+   3. The competitor limit is per competition. Per day competitor limits are not allowed.
+   4. The WCA Competition Announcement Team will review the competitor limit and/or qualification results or rankings to ensure that they are utilized properly.
+6. Registration requirements
+   1. The registration requirements must be clear so that any competitor from anywhere in the world knows how to and has the ability to register.
+      1. The registration requirements must be available in English.
+      2. In addition to the English version, the registration requirements can be available in any other language and should be available in the local language.
+   2. The registration period(s) must be a clearly defined time range with a start and end time and date.
+      1. Each registration period must be announced publicly on the WCA website at least 48 hours before it starts.
+         1. The competition's WCA Delegate(s) and/or organizer(s) may register for the competition prior to the first registration period, however their registration(s) must not be approved until the first registration period has started.
+      2. Different registration periods may have different registration requirements (e.g. restricting which events a competitor can register for).
+   3. Different locations for multi-location competitions may have different registration requirements (e.g. competitor limit).
+7. Events
+   1. Each event must have a planned number of rounds.
+      1. Each round of each event must have a planned format and time limit.
+      2. A round of an event must have planned advancement conditions if there is a subsequent round of the event.
+      3. A round may have a cutoff. A round with a cutoff is a combined round, with the formats as per [WCA Regulation 9b)](wca{regulations/#9b}).
+   2. Only results from events that appear on the competition website in the Events tab and Schedule will be recognized.
+      1. Other events may only be held unofficially.
```

Comment:
I agree with that addition. Two comments, though:

1. Should we change the `should` to a `must`?
2. The indentation looks wrong.

Reply:
> I agree with that addition. Two comments, though:
> 
> 1. Should we change the `should` to a `must`?
> 2. The indentation looks wrong.

I think using 'should' will be good right now. This gives a bit of grace period for delegates who have the potential to make a mistake. I wouldn't consider it right to deny a main event (which realistically only hurts the podium of the main event) due to this mistake at this point in time. I think it may be something to change at a later version (maybe 6months to a year from this document's announcing).

Reply:
Thanks, LGTM now! I'm committing it as is.